CHANGELOG/1.2-CHANGELOG.md

```diff
@@ -0,0 +1,424 @@
+# v1.2 Changelog
+
+## Table of Contents
+
+- [v1.2.0-rc1](#v120-rc1)
+
+# v1.2.0-rc1
+
+We expect that this release candidate is quite close to the final v1.2.0
+release. However, subsequent breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but we do not recommend
+shipping products based on a release candidate API due to the possibility of
+incompatible changes prior to the final release. The following represents the
+changes since v1.1.0:
+
+## Release Cycle changes
```

Comment:
Non-blocking nit, feel free to ignore: I feel like this could probably be further down the list as most readers will probably be more interested in the new user-facing features.

Reply:
I actually put this first on purpose so that folks who only look for features will end up looking at it more, so I'd rather leave it.